sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ outputs = asyncio.run(custom_simulator(
 
 ### Breaking Changes
 - Renamed environment variable `PF_EVALS_BATCH_USE_ASYNC` to `AI_EVALS_BATCH_USE_ASYNC`.
+- `RetrievalEvaluator` now requires a `context` input in addition to `query` in single-turn evaluation.
```

Comment:
Please help me understand what's the difference between input `context` and the global `context` in `conversation` object.

Reply:
Conversations only have one system/global context that's applied to all turns whereas single-turn conversations have their own individual context.

It's the same as we currently do for `GroundednessEvaluator` and `CoherenceEvaluator`.